README.md

```diff
@@ -32,6 +32,14 @@ Altenatively one can use the embedding API, see [src/mount.coffee](./src/mount.c
     rt.start (err, rt) ->
       # started
 
+With NoFlo 0.7.0, component discovery and caching 
```

Comment:
Should have a headline. "Component cache" ? Needs a sentence about the benefit (reason to use this): faster process startup due to less disk IO.
Or maybe put section at the end as "Optimizing startup time" ? Then also is a place we can mention bundling with Webpack